glide.lock

```diff
@@ -23,9 +23,14 @@ imports:
 - name: github.com/m3db/m3cluster
   version: b596a86b06f37c9150ea352d6ec1549ed19f304b
   subpackages:
+  - client
```

Comment:
Yea, I have no idea why glide changed anything :( All I did was change the dependency and run `glide update -v`.